elliottlib/bzutil.py

```diff
@@ -1063,8 +1063,18 @@ def is_first_fix_any(bugtracker, flaw_bug, current_target_release):
         return True
 
     # filter tracker bugs by OCP product
-    tracker_bugs = [b for b in bugtracker.get_bugs(tracker_ids)
-                    if b.product == constants.BUGZILLA_PRODUCT_OCP and b.is_tracker_bug()]
+    tracker_bugs = []
+    for tracker_id in tracker_ids:
+        try:
+            b = bugtracker.get_bug(tracker_id)
+        except Exception as e:
+            # first-fix tracker bug might be not visible but not break here
+            logger.info("Failed to get tracker bug info from bugzilla API, need manually check if it's permission issue")
```

Comment:
`info` is not obvious to release artists. It should be a warning.
Also please include the bug ID and the original exception message in the log message.

Reply:
updated